Sources/SwiftProtobuf/UnknownStorage.swift

```diff
@@ -44,8 +44,3 @@ public struct UnknownStorage: Equatable {
     }
   }
 }
-
-#if swift(>=5.5) && canImport(_Concurrency)
-// Once our minimum supported version has Data be Sendable, @unchecked could be removed.
-extension UnknownStorage: @unchecked Sendable {}
-#endif
```

Comment:
Do we need to revert this?

Reply:
One of the unit tests has a hand-written conformance to `Message` that isn't code-generated. This was added to allow that type to be `Sendable`, because that had become a requirement of `Message`. The workaround is no longer needed if we're not changing the API of the runtime library for the moment.

Reply:
I was thinking that we know we want all our types to be Sendable to the extent it's possible, even if the `Message` and `Enum` protocols cannot be made so just yet without breaking compatibility.

Reply:
> to the extent it's possible

Therein lies the rub. I agree with you that the support library should express `Sendable` in as much public API as it can, and it can't for almost all of its public API because of `Data`. I think leaving just the one conformance is misleading. It implies the support library has been otherwise audited, which it hasn't.

Reply:
I would like to leave comments in the library near `Message`, `Enum`, and `UnknownStorage`:
```
// TODO: This should be `Sendable` but we cannot do so yet without possibly breaking compatibility.
```
That will reduce the likelihood we miss these when we are able to break compatibility.

Reply:
Will do ASAP.

Reply:
(without having double checked the raw code), I think the hand written code was actually a copy from one point in time to also ensure things still worked without regenerating.  So UnknownStorage might not need this if we do the version bump because that whole test with the copy of the generated code might be obsolete.